yarn.lock

```diff
@@ -10431,11 +10386,16 @@ typedarray@^0.0.6:
   resolved "https://registry.yarnpkg.com/typedarray/-/typedarray-0.0.6.tgz#867ac74e3864187b1d3d47d996a78ec5c8830777"
   integrity sha1-hnrHTjhkGHsdPUfZlqeOxciDB3c=
 
-typescript@^3.3.0, typescript@^3.7.2:
+typescript@^3.3.0:
```

Comment:
We now have two versions of typescript in use? Should there only be one? (the other looks to be bedrock)

Reply:
Having one version sounds reasonable. How do we do this?

Reply:
Two options that I know of:
1) Manually edit the lock file (not a good idea)
2) Re-add bedrock and that should make it use the new version.

Reply:
I've done #2. Thanks!

Reply:
Now we have two webpacks 🤣

Reply:
Fixed.